database/mssql/scripts/dbscripts/PSP_PIMS_S15_01/Test Data/04_DML_LEASE_LOAD.sql

```diff
@@ -0,0 +1,318 @@
+/* -----------------------------------------------------------------------------
+Insert leases for testing
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2021-Aug-19  Initial version
+----------------------------------------------------------------------------- */
+
+-- Insert the lease data
+INSERT INTO [dbo].[PIMS_LEASE]([LEASE_PURPOSE_TYPE_CODE], [LEASE_PURPOSE_SUBTYPE_CODE], [LEASE_STATUS_TYPE_CODE], [LEASE_PMT_FREQ_TYPE_CODE], [LEASE_PROGRAM_TYPE_CODE], [TENANT_ID], [TFA_FILE_NO], [PS_FILE_NO], [START_DATE], [RENEWAL_DATE], [EXPIRY_DATE], [LEASE_AMOUNT], [INSURANCE_START_DATE], [INSURANCE_END_DATE], [SECURITY_START_DATE], [SECURITY_END_DATE], [INSPECTION_DATE], [INSPECTION_NOTES], [LEASE_NOTES], [UNIT], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_DIRECTORY])
```

Comment:
It looks like PSP_PIMS_S15_00 inadvertently overwrote all of my updates to these LEASE test data scripts again. This appears to be an older version of this script that won't run against the current schema. 

How can I better communicate those changes to you? I brought this up in a previous PR but it still seems to be a problem.

My changes are in PSP_PIMS_S13_00

Reply:
@dfilteau

Reply:
The Test Data folder was in there for reference and I didn't expect it to be used currently.  We should extract the test data you're using and capture it in this folder so it can be easily rolled out anywhere.